data/json/mutations/mutations.json

```diff
@@ -7058,5 +7058,13 @@
     "description": "You have fast reflexes, allowing you to dodge attacks and grabs more easily.",
     "starting_trait": true,
     "dodge_modifier": 2
+  },
+  {
+    "type": "mutation",
+    "id": "HAS_NEMESIS",
+    "name": { "str": "Hunted" },
+    "points": -2,
+    "description": "Since the start of the Cataclysm, something has been hunting you relentlessly. You can outrun it, but it always seems to find you eventually. The only escape is for one of you to die.",
```

Comment:
```suggestion
    "description": "Since the start of the Cataclysm, something has been hunting you relentlessly.  You can outrun it, but it always seems to find you eventually.  The only escape is for one of you to die.",
```